docs/dev/building-on-zksync/hello-world.md

```diff
@@ -1,80 +1,66 @@
 # Quickstart
 
-In this quickstart guide, you will learn how to deploy a smart contract to zkSync and build a dApp to interact with it using the zkSync development toolbox.
-
-This is what we're going to build:
-
-- A smart contract that stores a greeting message and is deployed on zkSync.
-- A dApp to retrieve and update the greeting message.
-- Users will be able to change the greeting on the smart contract.
-- By default, users will pay transaction fees in ETH. However, we will also explain how to [implement the testnet paymaster](#paying-fees-using-testnet-paymaster) to allow users to pay transaction fees with ERC20 tokens.
-
+This guide shows you how to deploy a smart contract to zkSync and build a dApp that interacts with it using the zkSync development toolbox.
 
+This is what we're going to do:
 
+- Build and deploy a smart contract on zkSync that stores a greeting message.
+- Build a dApp that retrieves and updates the greeting message.
+- Allow users to change the greeting message on the smart contract via the app.
+- Show you how to [implement the testnet paymaster](#paying-fees-using-testnet-paymaster) that allows users to pay transaction fees with ERC20 tokens instead of ETH.
 
 ::: warning
-
-Please note that breaking changes were introduced in `zksync-web3 ^0.13.0`. The API layer now operates with `gas` and the `ergs` concept is only used internally by the VM.
-
+- Breaking changes were introduced in `zksync-web3 ^0.13.0`. 
+- The API layer now operates with `gas` and the `ergs` concept is only used internally by the VM.
```

Comment:
Let's remove this warning message.
It's not relevant anymore, this change is already a known fact, it occurred over a month ago.

Reply:
DONE